Tests/FileChangeCheckerTests.cs

```diff
@@ -22,5 +25,45 @@ public void CheckIfFileCheckerHashesCorrectly()
             FileChangeChecker.FileIsModified(original, modified).Should().Be(true);
 
         }
+        [Fact]
+        public void CheckIfProjectIsModifiedOutputsCorrectValue()
+        {
+            var mock = new Mock<IDirectory>();
+
+            var fakeDirectory = mock.Object;
+
+            List<String> fakePaths = new List<String>() { "hi", "hello" };
+
+            List<Stream> fakeList = new List<Stream>();
+
+            List<Stream> fakeList2 = new List<Stream>();
+
+            for (int i = 0; i < fakePaths.Count; i++)
+            {
+                var fakeStream = new MemoryStream();
+                fakeStream.Write(Encoding.ASCII.GetBytes("thing " + i + "that gets hashed"));
+                fakeStream.Position = 0;
+                fakeList2.Add(fakeStream);
+
+            }
+            for (int i = 0; i < fakePaths.Count; i++)
+            {
+                var fakeStream = new MemoryStream();
+                fakeStream.Write(Encoding.ASCII.GetBytes("thing " + i + "that gets hashed"));
+                fakeStream.Position = 0;
+                fakeList.Add(fakeStream);
+
+            }
+
```

Comment:
These 2 `for` loops are identical apart from the list that they add to. I would extract a method something like `CreateFakeStream(int i)` and call it here, so the code is like:
```
for (int i = 0; i <fakePaths.Count; i++)
{
    fakeList.Add(CreateFakeStream(i));
    fakeList2.Add(CreateFakeStream(i));
}
```